reddit_service_ads_tracking/__init__.py

```diff
@@ -0,0 +1,140 @@
+import base64
+import logging
+import json
+import time
+
+from baseplate import (
+    Baseplate,
+    make_metrics_client,
+)
+from baseplate.crypto import (
+    ExpiredSignatureError,
+    IncorrectSignatureError,
+    MessageSigner,
+    UnreadableSignatureError,
+)
+from baseplate.integration.pyramid import BaseplateConfigurator
+from enum import Enum
+from pyramid.config import Configurator
+
+from reddit_service_ads_tracking import config
+from reddit_service_ads_tracking.lib import (
+    events,
+    response,
+    urls,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+class ClickProcessNotes(Enum):
+    VALID = "VALID"
+    EXPIRED_SIGNATURE = "EXPIRED_SIGNATURE"
+    INVALID_SIGNATURE = "INVALID_SIGNATURE"
+    BAD_DATA = "BAD_DATA"
+
+
+class TrackingService(object):
+    def __init__(self, config, signer, event_queue):
+        self.config = config
+        self.signer = signer
+        self.event_queue = event_queue
+
+    def is_healthy(self, request):
+        return {
+            "status": "healthy",
+        }
+
+    def track_conversion(self, request):
+        pass
+
+    def track_impression(self, request):
+        pass
+
+    def track_click(self, request):
+        url = request.GET.get("url", "").encode("utf-8")
+        b64_data = request.GET.get("data", "").encode("utf-8")
+        observed_mac = request.GET.get("hmac", "").encode("utf-8")
+
+        if (not url or
+                not b64_data or
+                not observed_mac):
+            return response.abort(request, 400,
+                                  "missing required query parameters")
+
+        result, process_notes = None, ClickProcessNotes.VALID
```

Comment:
can this just be written on two lines? feels a bit odd to have it tuplized like this (harder to read)